cranelift/assembler-x64/meta/.rustfmt.toml

```diff
@@ -1,5 +1,5 @@
 # This extra configuration allows defining extra-long lines in
 # `src/instructions`.
 fn_call_width = 100
-max_width = 110
+max_width = 120
```

Comment:
It could be helpful to review this commit separately since it contains all the formatting changes (`cargo fmt`) and no functional changes.